databuilder/databuilder/models/application.py

```diff
@@ -21,88 +21,51 @@
 from databuilder.utils.atlas import AtlasRelationshipTypes, AtlasSerializedEntityOperation
 
 
-class Application(GraphSerializable, TableSerializable, AtlasSerializable):
+class GenericApplication(GraphSerializable, TableSerializable, AtlasSerializable):
```

Comment:
I think just calling it Application is generic enough

Reply:
+1, do we need to change the model name?

Reply:
Ah yes, I wish that was an option. This is being done for bw-compatibility. The Airflow-specific class was/is called `Application` and I have kept that working via https://github.com/amundsen-io/amundsen/pull/1577/files#diff-9ecb77ae83ba63ef4784084c73ecfd0be606ba36d7dcf640e424d9f38dfc3fc6R240.

Reply:
In other words, 
- `GenericApplication` is completely new class
- `Application` class is renamed `AirflowApplication`
- `AirflowApplication` is aliased as `Application` for compatibility